app/frontend/src/components/Example/ExampleList.tsx

```diff
@@ -1,28 +1,40 @@
+import { useEffect, useState } from "react";
 import { Example } from "./Example";
-
+import { DEFAULT_EXAMPLES, GPT4V_EXAMPLES } from "../../i18n/examples.js";
 import styles from "./Example.module.css";
-
-const DEFAULT_EXAMPLES: string[] = [
-    "What is included in my Northwind Health Plus plan that is not in standard?",
-    "What happens in a performance review?",
-    "What does a Product Manager do?"
-];
-
-const GPT4V_EXAMPLES: string[] = [
-    "Compare the impact of interest rates and GDP in financial markets.",
-    "What is the expected trend for the S&P 500 index over the next five years? Compare it to the past S&P 500 performance",
-    "Can you identify any correlation between oil prices and stock market trends?"
-];
+import { setInterval as workerSetInterval, clearInterval as workerClearInterval } from "worker-timers";
```

Comment:
Is it necessary to use worker-timers versus standard setInterval/clearInterval? I would think even that shuffling upon display would be fairly effective.
I'm trying to avoid adding more to package.json when possible.